src/components/VContentSwitcher/VContentSwitcherButton.vue

```diff
@@ -52,13 +57,28 @@ export default {
     const isIconButton = computed(
       () => isHeaderScrolled?.value && !isMinScreenMd.value
     )
-    const sizeClasses = computed(() =>
-      isIconButton.value ? 'w-10 h-10' : 'ps-2 pe-3'
-    )
+    const sizeClasses = computed(() => {
+      if (props.type === 'searchbar') {
+        return 'h-12 px-2'
+      } else if (isIconButton.value) {
+        return 'w-10 h-10'
+      } else if (isMinScreenMd.value) {
+        // When there is a caret down icon, paddings are balanced
+        return 'px-2'
+      } else {
+        // Without caret down icon, paddings need to be adjusted
+        return 'ps-2 pe-3'
+      }
```

Comment:
Seems that the last two conditions can be combined and simplify the code.
```suggestion
      } else {
        /**
          When there is a caret down icon (on 'md' screens), paddings are balanced, 
          without it, paddings need to be adjusted.
          */
        return 'ps-2 pe-3 md:px-2'
      }
```